extensions/notebook/src/book/bookEdit.ts

```diff
@@ -9,23 +9,23 @@ import * as yaml from 'js-yaml';
 import * as vscode from 'vscode';
 import * as loc from '../common/localizedConstants';
 import { BookTocManager } from './bookTocManager';
-import { IBookUndoRedoElement } from './bookUndoRedoService';
+import { IUndoRedoElement } from './undoRedoService';
 
-export interface undoRedoToc {
-	undo: JupyterBookSection[],
-	redo: JupyterBookSection[]
+export interface UndoRedoState {
```

Comment:
I'd still suggest calling this `BookUndoRedoState` - since this is specific to books.